lib/query/stored.js

```diff
@@ -65,6 +65,15 @@ const renameStoredQuery = (conn, name, newName) => {
   }).then(httpBody);
 };
 
+const retrieveStoredQuery = (conn, name, options = {}) => {
```

Comment:
```suggestion
const getStoredQuery = (conn, name, options = {}) => {
```

also move above `list`